lib/config/types.go

```diff
@@ -48,11 +49,17 @@ func (p Project) SourceRepo() Repo {
 // A RepositoryType describes a type of revision control system which manages target revisions of deployment.
 type RepositoryType string
 
+type HostType string
```

Comment:
@hokichaio great implementation!! could we have small explanation for this variable ? :pray: